browser.css

```diff
@@ -230,3 +230,32 @@ html.os-win32 .back-btn { top: 0; }
   border: none;
   margin: 0;
 }
+
+body {
```

Comment:
let this app can drag. because when I scroll to the bottom, this app can not drag. you can try it.